### PR TITLE
April 2024 monthly blog post

### PR DESCRIPTION
**Pull request by sarah11918**
Adds the Apr 30 blog post covering the month of April

### CODE REVIEW

**Comment by liruifengv** [comment]
image

The screenshot generate service `screenshot.11ty.dev` does not support Chinese characters...

**Comment by sarah11918** [comment]
Thank you, @liruifengv ! Chris has a proposed solution for a different process, but we won't be able to switch to using it today. So, I will keep this site back and post it *next* month instead, when we should no longer have the problem!

File: src/content/blog/whats-new-april-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+lang: "en"
+---
+import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+
+Some exciting news from around the Astro ecosystem this month. Let's dive in!
+
+## Updates from Mission Control
+
+- 🥳 Astro corsses **10,000,000** downloads on NPM!
```
Comment (HiDeoo) [inline src/content/blog/whats-new-april-2024.mdx:18]:
```suggestion
- 🥳 Astro crosses **10,000,000** downloads on NPM!
```

I think this may be a typo?

**Review by dreyfus92 — APPROVED** [review]
Everything looks great! 🫡

**Pull request merged**